shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1211,7 +1211,7 @@ object GlutenConfig {
       .internal()
       .doc("The IO threads for connector split preloading")
       .intConf
-      .createWithDefault(0)
+      .createWithDefaultFunction(() => SQLConf.get.getConfString("spark.executor.cores", "1").toInt)
```

Comment:
may I ask why use spark.executor.cores as default values? why not set 2 to align with Velox?